chroma-manager/tests/framework/utils/cluster_setup.sh

```diff
@@ -38,7 +38,7 @@ for key in CentOS-7 redhat-release; do
     fi
 done
 
-if [[ \$HOSTNAME = *vm*2 ]] || [[ \$HOSTNAME = $CLIENT_1 ]]; then
+if [[ \$HOSTNAME = $CLIENT_1 ]]; then
```

Comment:
Rather than aborting various test runs (as above) due to a lack of `$CLIENT_1` in the cluster configuration (and requiring it to be present here)  I think the more conservative and flexible approach is to just add a `[ -n "$CLIENT_1"]` to this `if` statement.  That would keep this file flexible enough to use with tests that don't require a client.